classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
```

Comment:
- [ ] When adding a student to a classroom it also sets the classroom for the student. It will be nice to fix this by adding this: `student.classroom = self` below line 11.